src/plugins/embeddable/public/index.ts

```diff
@@ -83,6 +85,8 @@ export function plugin(initializerContext: PluginInitializerContext) {
   return new EmbeddablePublicPlugin(initializerContext);
 }
 
+export const [getTheme, setTheme] = createGetterSetter<ThemeServiceStart>('Theme');
```

Comment:
Same here, I think we shouldn't export these helpers from a plugin. They should be hidden inside plugin. 
Applied for all the plugin where we export these from `public/index.ts`

Reply:
moved to services file